### PR TITLE
fix: tokyo night

### PR DESCRIPTION
**Pull request by caarlos0**
refs #318 
refs #229 

it actually needs to be in Go code. JSON is generated from Go, not sure why.

### CODE REVIEW

**Comment by bashbunni** [comment]
Sorry for the confusion! This should now be in the right state

File: styles.go [hunk 1]
```diff
@@ -642,17 +642,224 @@ var (
 		HTMLSpan:  ansi.StyleBlock{},
 	}
 
+	// TokyoNightStyle is the tokyo night style.
```
Comment (aymanbagabas) [inline styles.go:645]: Should we move `dracula.go` here as well?

**Review by aymanbagabas — APPROVED** [review]

File: styles.go (hunk 1 above)
Comment (bashbunni) [reply styles.go:645]: Ah I see what you mean. I think so. We can re-organize them another time if this file gets too big, but if all of our themes are there except Dracula, then yeah def worth moving it to that file. I'll move it over :)

Comment (aymanbagabas) [reply styles.go:645]: I can see `styles.go` growing so large that way. Perhaps we could move `dracula.go` to `style_dracula.go`, wdyt?

Comment (aymanbagabas) [reply styles.go:645]: Related: https://github.com/charmbracelet/glamour/issues/298

Comment (bashbunni) [reply styles.go:645]: kk I'll play with it a bit to see how we can clean it up better

**Comment by bashbunni** [comment]
Moved to the `styles` package, they all still work as expected. Added another example that reads the environment for easier testing of custom styles. It's under `stdin-stdout-custom-styles` :)

Dracula theme:
image

Tokyo Night:
image

Default:
image

File: styles/styles.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package glamour
+package styles
 
 //go:generate go run ./internal/generate-style-json
```
Comment (aymanbagabas) [inline styles/styles.go:3]:
```suggestion
//go:generate go run ../internal/generate-style-json
```

**Comment by bashbunni** [comment]
fwiw style test is failing due to the name being case sensitive
image

**Pull request merged**